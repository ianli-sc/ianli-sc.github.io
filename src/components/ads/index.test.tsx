import { render } from '@testing-library/react';

import React from 'react';
import Ads from './index';
 
test('renders ads by snapshot', () => {
  const { container } = render(<Ads />);
  expect(container).toMatchSnapshot();
});