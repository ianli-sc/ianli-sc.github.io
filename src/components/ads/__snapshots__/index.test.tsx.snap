// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ads by snapshot 1`] = `
<div>
  <div
    class="ads"
  >
    <div
      class="text-inline"
    >
      BLACK FRIDAY SALE | Up to 50% off FW22 with an extra 10% off using code ‘BLACK10’ at checkout. 
      <span
        class="link"
      >
        T&Cs apply
      </span>
    </div>
  </div>
</div>
`;
